iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/TopicParser.java

```diff
@@ -3,7 +3,7 @@
 
 package com.microsoft.azure.sdk.iot.device.transport.mqtt;
 
-class TopicParser
+public class TopicParser
```

Comment:
mark the class and methods `public` as they are being used for both "mqtt"(v3) and "mqtt5"